scripts/build_ngx_pagespeed.sh

```diff
@@ -0,0 +1,331 @@
+#!/bin/bash
+
+# Put everything in one big function so that if this script is executed after a
+# partial download it doesn't do anything.
+function build_ngx_pagespeed() {
+  getopt --test
+  if [ "$?" != 4 ]; then
+    echo "Your version of getopt is too old.  Exiting with no changes made."
+    # Even Centos 5 and Ubuntu 10 LTS have new-style getopt, so I don't expect
+    # this to be hit in practice on systems that are actually able to run
+    # PageSpeed.
+    exit 1
+  fi
+
+  # If we set -e or -u then users of this script will see it silently exit on
+  # failure.  Instead we need to check the exit status of each command
+  # manually.  The run function (see below) handles exit-status checking for
+  # system-changing commands.
+
+  function usage() {
+    echo "
+Usage: build_ngx_pagespeed.sh [options]
+Options:
+  -v, --version <ngx_pagespeed version>
+      What version of ngx_pagespeed to build.  Required.
+
+  -n, --nginx <nginx version>
+      What version of nginx to build.  If not set, this script only prepares the
+      ngx_pagespeed module, and expects you to handle including it when you
+      build nginx.
+
+  -b, --builddir <directory>
+      Where to build.  Defaults to \$HOME.
+
+  -p, --depsinstalled
```

Comment:
Wow this is weirdly named. What about --no_deps_check & corresponding variable $DO_DEPS_CHECK (defaults to true)?